### PR TITLE
Feat/script-migration

### PR DESCRIPTION
**Pull request by 0xtekgrinder**

### CODE REVIEW

File: utils/getConnectedChains.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const { registry, ChainId } = require('@angleprotocol/sdk');
```
Comment (GuillaumeNervoXS) [inline utils/getConnectedChains.js:1]: We should directly put it in the utils repo

File: utils/getConnectedChains.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+const { ethers } = require('ethers');
+
+if (process.argv.length < 3) {
+    console.log('Usage: node getConnectedChains.js <chainId>');
+    process.exit(1);
+}
+
+const contract = process.argv[2];
+
+// Try to parse the input as a number if possible
```
Comment (GuillaumeNervoXS) [inline utils/getConnectedChains.js:11]: ?

File: scripts/proposals/angleTokens/ConnectAgTokenSideChainMultiBridge.s.sol [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+contract ConnectAgTokenSideChainMultiBridge is Wrapper {
+    SubCall[] private subCalls;
+
+    function run() external {
+        uint256 chainId = vm.envUint("CHAIN_ID");
+
+        /** TODO  complete */
+        string memory description = vm.envString("DESCRIPTION");
+        address token = vm.envAddress("TOKEN");
```
Comment (GuillaumeNervoXS) [inline scripts/proposals/angleTokens/ConnectAgTokenSideChainMultiBridge.s.sol:19]: the 2 (token, lzToken) should be fetched from the sdk

File: helpers/createAgTokenSideChainMultiBridge.sh [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    if [ $? -ne 0 ]; then
+        echo ""
+        echo "Deployment failed"
+        exit 1
+    fi
+
+    echo ""
+    echo "Deployment successful"
+
+    echo ""
```
Comment (GuillaumeNervoXS) [inline helpers/createAgTokenSideChainMultiBridge.sh:51]: Let's add test files to check that everything is well setup and do bridge transaction by faking LZ

File: helpers/createChain.sh [hunk 5]
```diff
@@ -0,0 +1,49 @@
+#! /bin/bash
+
+source lib/utils/helpers/common.sh
+
+function usage {
+  echo "bash createChain.sh <chain> <chainUri>"
+  echo ""
+  echo -e "chainId: chain to run the script on"
+  echo -e "chainUri: rpc uri for the chain"
```
Comment (GuillaumeNervoXS) [inline helpers/createChain.sh:9]: Same shouldn't be provided

File: helpers/createTokenSideChainMultiBridge.sh [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+        exit 1
+    fi
+
+    echo ""
+    echo "Deployment successful"
+
+    echo ""
+    echo "Would you like to create the proposal for the token side chain multi bridge ? (y/n)"
+
+    read createProposal
```
Comment (GuillaumeNervoXS) [inline helpers/createTokenSideChainMultiBridge.sh:85]: Same let's add test before

**Comment by 0xtekgrinder** [comment]
Closing this one for now it has been instead converted into angle-multisig repository

**Pull request closed without merging**